### PR TITLE
make README a markdown document

### PR DESCRIPTION
**Pull request by champignoom**
Also mention explicitly the yaml version supported (1.1)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
-PyYAML - The next generation YAML parser and emitter for Python.
+## PyYAML - The next generation YAML 1.1 parser and emitter for Python
```
Comment (ingydotnet) [inline README.md:1]: I would prefer:
```
PyYAML
======

The next generation YAML 1.1 parser and emitter for Python
```

**Review by ingydotnet — COMMENTED** [review]
Thanks for the PR.

Since this changes a file name, I am moving it to [PyYAML 6.0 Release Planning](https://github.com/yaml/pyyaml/projects/7).

I'm planning more file name changes there, and would rather do them at once to avoid potential churn on downstream packaging.

**Comment by bsolomon1124** [comment]
https://github.com/yaml/pyyaml/pull/327 does this as well.  (While https://github.com/yaml/pyyaml/pull/186 converts it to RST.)

**Comment by bsolomon1124** [comment]
> Since this changes a file name, I am moving it to PyYAML 6.0 Release Planning.

If that type of backwards compatibility is a concern, could also symlink `README` to `README.md`.

**Comment by Thom1729** [comment]
It looks like this was included in https://github.com/yaml/pyyaml/commit/702b1767bf632d5bae390ec1f855173d256bb52f.

**Pull request closed without merging**